freetextresponse/public/view.less

```diff
@@ -111,5 +111,50 @@
     font-size: 0.9em;
   }
 
+  .responses-box {
+    display: none;
+    margin-top: 10px;
+    background: #f8f8f8;
+    border: 2px solid #ddd;
+    padding: 10px 10px 0px 10px;
+  }
+
+  .responses-title {
+    margin-bottom: 5px;
+  }
+
+  .hide-button {
+    float: right;
+    background: none;
+    border: none;
+    padding: 0;
+    color: #069;
+    text-decoration: underline;
+    cursor: pointer;
+    font-size: 0.8em;
+  }
+
+  .response-list {
+    list-style: none;
+    text-align: center;
+  }
+
+  .other-student-responses {
+    display: inline-block;
+    background: white;
```

Comment:
do we NEED to set a background color? if we don't have to, it's better not to, so we avoid the white on off-white, like in the forum...